private-identity-audit/ffi/examples/c_example.c

```diff
@@ -0,0 +1,86 @@
+#include <stdio.h>
+#include <stdint.h>
+#include <inttypes.h>
+#include <string.h>
+#include "c_example.h"
+
+/**
+ * Creates a CDD ID from a CDD claim.
+ * (copy/pasted here from confidential-identity/ffi/examples/c_example.h
+ *  for convenience.)
+ *
+ * SAFETY: Caller is responsible to make sure `cdd_claim` pointer is a valid
```

Comment:
Please run clippy on these files. Clippy requires a specific format for the "safety" messages.

Reply:
What did you run to see that formatting issue? just `cargo clippy`? It gives me irrelevant errors.

Reply:
nvm